csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -625,10 +686,9 @@ void HopperMultipleMatmulScheduler::scheduleEpilogue() {
       dc->setMemoryType(MemoryType::Local);
       d_smem->setMemoryType(MemoryType::Shared);
 
-      auto store_with_stmatrix = dataTypeSize(dc->dtype()) == 2;
-
+      // Set LoadStoreOp
+      bool store_with_stmatrix = dataTypeSize(dc->dtype()) == 2;
```

Comment:
```suggestion
      bool store_with_stmatrix = params_->use_stmatrix && dataTypeSize(dc->dtype()) == 2;
```

Should we also disable ldmatrix whenever stmatrix is disabled? And should we rename that parameter if so? In either case we probably should update the comment in `matmul_heuristic.h`.

Reply:
Renamed `use_stmatrix` to `use_ldst_matrix`.

Updated comment:
```cpp
  //! If use_smem_epilogue==false, this has no effect. Otherwise, it enables
  //! storing the mma result to shared memory using stmatrix and loading
  //! epilogue inputs to registers using ldmatrix instructions. Note that
  //! stmatrix nor ldmatrix are never used on TensorViews whose dtype is fp16
  //! or bf16.
  bool use_ldst_matrix = true;
```